_data/vishnudut/vishnudut.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Vishnu Dut V
+institution: VIT Chennai
+profile_pic: vishnu.jpg
+profile_pic: Vishnu.jpg
```

Comment:
Hey @vishnudut , this is a redundant line. Consider removing it

Reply:
Also the profile_pic name should be exact as per the picture file name. Follow the exact format as per the template